bin/git/update-hooks

```diff
@@ -12,6 +12,43 @@ HOOK_DIR="$(git rev-parse --show-toplevel)/.git/hooks"
 
 . "$SRC_DIR/hook_lib"
 
+usage() {
+    echo "Usage: update-hooks [OPTION]"
+    echo "Update the hooks in the local git repository to match those"
+    echo "under version control."
+    echo
+    echo "Options:"
+    echo "  --dry-run       Do not make any updates"
+    echo "  -v  --verbose   Verbose output, including diffs of updated files"
+    echo "  --help          Display this help text"
+}
+
+# Flags that can be set from the command line, with default values
+make_updates=1  # the inverse of --dry-run
+verbose=0
+
+# parse command line options
+while [ "$#" -gt 0 ]; do
+    case "$1" in
+        (--help)
+            usage
+            exit 0
+            ;;
+        (-v|--verbose)
+            verbose=1
+            ;;
+        (--dry-run)
+            make_updates=0
+            ;;
+        (*)
+            echo "Unrecognized option: $1"
```

Comment:
Yes, of course.  I had meant to do that but it didn't make it from my brain to my editor.  Fixed in  9182afd